rules/windows/process_creation/proc_creation_win_hktl_mimikatz_command_line.yml

```
@@ -0,0 +1,54 @@
+title: HackTool - Mimikatz Execution
+id: a642964e-bead-4bed-8910-1bb4d63e3b4d
+status: test
+description: Detection well-known mimikatz command line arguments
+references:
+    - https://www.slideshare.net/heirhabarov/hunting-for-credentials-dumping-in-windows-environment
+    - https://tools.thehacker.recipes/mimikatz/modules
+author: Teymur Kheirkhabarov, oscd.community, David ANDRE (additional keywords), Tim Shelton
+date: 2019/10/22
+modified: 2023/02/21
+tags:
+    - attack.credential_access
+    - attack.t1003.001
+    - attack.t1003.002
+    - attack.t1003.004
+    - attack.t1003.005
+    - attack.t1003.006
+logsource:
+    category: process_creation
+    product: windows
+detection:
+    selection_main:
```

```suggestion
    selection_tools_name:
```
with main I expect a condition "main and selectionX"